api/src/domain/updateSidebarCards.ts

```diff
@@ -52,9 +52,7 @@ export const updateSidebarCards: UpdateSidebarCards = (userData: UserData): User
   }
 
   if (LookupOperatingPhaseById(operatingPhase).displayGoToProfileNudge) {
-    const isEveryOpportunityFieldAnswered = OPPORTUNITY_FIELDS.every((field) => {
-      return isFieldAnswered(field, currentBusiness.profileData);
-    });
+    const isEveryOpportunityFieldAnswered = isEveryFundingFieldAnswered(currentBusiness.profileData);
```

Comment:
[pebble] Let's keep the names consistent here. In our domain model, a Funding and an Opportunity is not the same thing. Opportunity is the umbrella term for both Fundings and Certifications. And the rest of the logic here should make it clear that we're really handling Opportunity fields here, so this new function name should reflect that.

Reply:
Overall, what's the reason for rewriting this logic in 2 different ways?

Here, you create a function called `isEveryFundingFieldAnswered` based on profileData.

In the `ProfileOpportunitiesAlert` we use two functions:
```
 getFieldsForProfile(state.profileData).filter((field) => {
    return isFieldAnswered(field, state.profileData);
  });
```

It seems like we're replicating the same logic in two different ways. And the `getFieldsForProfile` has some logical overlap with `isEveryFundingFieldAnswered` --- and `isEveryFundingFieldAnswered` isn't actually tested?

Can we just remove `isEveryFundingFieldAnswered` altogether and use the same logic in both places?

Reply:
done